src/test/java/com/karankumar/bookproject/backend/utils/PredefinedShelfUtilsTest.java

```diff
@@ -162,8 +156,8 @@ void shouldGetAllBooksInChosenShelves() {
         Set<Book> actualBooks = predefinedShelfUtils.getBooksInPredefinedShelves(predefinedShelves);
 
         // then
-        assertEquals(expectedBooks.size(), actualBooks.size());
-        assertTrue(actualBooks.containsAll(expectedBooks));
+        assertThat(actualBooks).hasSize(expectedBooks.size());
+        assertThat(actualBooks).containsAll(expectedBooks);
```

Comment:
I know this wasn't there before, but could you please combine these using `assertSoftly`?